sample_cgfs/rv32i_priv.cgf

```diff
@@ -1,21 +1,150 @@
 ecall:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ecall: 0
 
 ebreak:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ebreak: 0
+  
+misa:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); def rvtest_mtrap_routine=True; def rvtest_strap_routine=True
+  csr_comb:
+    misa >> (xlen-2) == 0x01 : 0
+    (misa >> 30) & 0x3 == 0x3 : 0
+    misa & 0x30 == 0x30 : 0
+    misa & 0x04 == 0x01 : 0
+    misa & 0x08 == 0x01 : 0
+    misa & 0x03 == 0x01 : 0
+    misa & 0x05 == 0x01 : 0
+
+mvendorid:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); def rvtest_mtrap_routine=True; def rvtest_strap_routine=True
+  mnemonics: 
+    csrrw: 0
+    csrrs: 0
+  csr_comb:
+    mvendorid & 0x80 == 0x80 : 0
```

Comment:
This is not applicable for all implementations. It is possible for an implementation to have `mvendorid` hardwired to `0`.